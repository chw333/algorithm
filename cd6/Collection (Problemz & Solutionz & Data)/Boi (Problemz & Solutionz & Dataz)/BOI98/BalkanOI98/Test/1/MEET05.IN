203 29 121
11 1 5
21 19 6
25 2 1
25 12 5
34 13 1
35 29 3
37 14 2
44 30 9
46 9 7
48 32 7
50 39 4
52 51 5
53 9 6
54 3 3
58 2 7
58 20 9
59 1 4
59 16 7
60 3 8
65 49 2
68 8 4
69 59 9
70 61 6
72 29 6
73 70 5
74 4 7
76 17 5
77 25 2
79 3 9
80 66 7
81 46 8
81 71 2
83 10 6
83 79 10
84 52 7
85 36 4
88 47 8
89 57 9
92 26 3
94 78 7
96 13 4
97 53 5
98 26 1
100 51 5
101 73 6
103 88 9
104 68 1
104 102 8
106 61 2
106 105 6
107 82 5
108 29 8
108 81 10
109 24 8
110 33 6
110 90 7
111 42 9
112 38 10
112 58 3
112 92 3
113 13 1
114 32 3
114 99 9
116 62 9
117 28 8
119 40 9
120 74 6
121 35 5
121 66 5
122 29 6
122 82 5
122 121 9
123 97 6
124 91 8
125 18 3
125 84 6
126 86 5
127 119 2
128 110 5
128 114 7
129 6 2
130 32 10
130 62 5
131 23 7
132 95 1
133 48 3
134 5 7
135 15 7
135 43 7
135 75 8
136 10 9
136 17 2
136 35 2
136 65 4
137 70 3
137 93 6
137 110 8
138 130 10
139 34 10
139 55 4
140 101 5
140 118 9
141 7 3
141 27 4
142 31 3
142 66 3
143 18 6
144 29 4
144 121 2
145 30 3
145 37 8
145 42 7
145 67 4
146 40 6
146 95 9
147 19 9
147 35 8
147 63 7
147 135 3
148 20 2
148 91 1
149 38 8
149 82 4
150 5 5
151 111 7
152 51 8
152 88 6
153 1 2
153 19 9
154 32 9
154 144 6
155 106 6
156 17 4
156 50 3
158 22 8
159 36 4
159 100 2
160 85 5
160 151 6
161 37 2
162 109 4
163 69 2
164 92 7
164 96 3
165 28 5
165 111 7
166 106 2
168 7 1
168 129 2
168 157 9
170 5 8
170 60 1
171 21 4
171 68 5
172 31 2
172 49 5
173 45 9
173 61 10
174 132 2
175 10 6
176 27 9
176 115 9
176 145 5
177 10 2
177 41 8
177 109 5
178 58 9
179 52 7
179 56 9
179 167 8
180 30 3
180 126 7
181 46 5
181 57 5
181 64 6
181 174 3
182 117 6
183 91 3
183 144 3
184 136 4
184 181 2
186 87 7
186 89 5
186 169 6
187 38 8
187 185 3
188 41 6
188 112 2
188 153 9
189 179 3
190 158 10
190 161 1
191 146 3
192 131 7
192 148 8
193 78 8
193 125 7
194 102 10
195 85 10
195 185 2
196 70 6
196 74 2
197 193 5
