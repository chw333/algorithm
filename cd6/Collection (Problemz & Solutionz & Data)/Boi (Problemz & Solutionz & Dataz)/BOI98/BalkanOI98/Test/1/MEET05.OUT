29 35 29
121 144 29
